hibernate-core/src/test/java/org/hibernate/jpa/test/lock/LockTest.java

```diff
@@ -61,6 +61,7 @@
 	private static final Logger log = Logger.getLogger( LockTest.class );
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done